### DISCUSSION TITLE
eject multistep form block

### DISCUSSION THREAD

**Discussion by Townieeee** [Budibase/budibase#13718] [Help] [answered]
Can one eject a multistep form block?
I just learnt how to eject a form block and the flexibility one has after that, but cannot see how to get the eject button to show with a multistep form block. Maybe not possible?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Townieeee,

There isn't an eject for the multi-step form. You can however still create a broken down version using the form component + steps components.

Feel free to create a feature request [here](https://github.com/Budibase/budibase/issues/new/choose).

Hopefully this helps.
